docs/performanceprofile/api-versions.md

```diff
@@ -2,18 +2,18 @@
 
 ## Supported API Versions
 
-The Performance Addon controller supports *v2*, *v1* and *v1alpha1* [Performance Profile](performance_profile.md) versions.
+The Performance Controller Operator supports *v2*, *v1* and *v1alpha1* [Performance Profile](performance_profile.md) versions.
```

Comment:
this is not a operator no more, so we should write "Performance Profile Controller" or something along these lines (here and below)

Reply:
done